src/curated-corpus/components/RemoveItemFromScheduledSurfaceModal/RemoveItemFromScheduledSurfaceModal.test.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { ScheduledCorpusItem } from '../../../api/generatedTypes';
+import { RemoveItemFromScheduledSurfaceModal } from './RemoveItemFromScheduledSurfaceModal';
+import { getTestApprovedItem } from '../../helpers/approvedItem';
+import { ScheduledSurfaces } from '../../helpers/definitions';
+
+describe('The RemoveItemFromScheduledSurfaceModal component', () => {
+  const approvedItem = getTestApprovedItem();
+  const item: ScheduledCorpusItem = {
+    approvedItem,
+    createdAt: 1635014926,
+    createdBy: 'jdoe',
+    externalId: '123-test-id',
+    scheduledDate: '2022-05-24',
+    scheduledSurfaceGuid: ScheduledSurfaces[0].guid,
+    updatedAt: 1635114926,
+    updatedBy: 'jdoe',
+  };
+  const isOpen = true;
+  const onSave = jest.fn();
+  const toggleModal = jest.fn();
+
+  it('should render this component successfully', async () => {
+    render(
+      <RemoveItemFromScheduledSurfaceModal
+        item={item}
+        isOpen={isOpen}
+        onSave={onSave}
+        toggleModal={toggleModal}
+      />
+    );
+
+    // fetching the modal heading and asserting it renders
+    expect(
+      screen.getByText(/remove this item from this scheduled surface/i)
+    ).toBeInTheDocument();
+
+    // fetching the form rendered within this modal by the form's title and asserting it gets rendered
+    expect(screen.getByText(item.approvedItem.title)).toBeInTheDocument();
```

Comment:
Once again disappointed with react-testing-library. I wanted to fetch the form that is rendered within this modal, using `getByRole()`, which is the correct way to do it. But, the testing library is unable to fetch it. I think it might be nested dom element problem?

In the screenshot below you can see the `<form ...` element is rendered with the correct properties but the testing library fails to fetch it.

![Screen Shot 2022-08-07 at 5 35 59 PM](https://user-images.githubusercontent.com/16694733/183301796-76ff1ea3-245e-414f-94ed-2c3aed0f6abb.png)

Reply:
Try dropping the `name` attribute on the query. It works for me - the modal is only pulling up one form, so it's not necessary here:

```typescript
const form = screen.getByRole('form');
expect(form).toBeInTheDocument();
```